### DISCUSSION TITLE
owner value is not changed in web UI

### DISCUSSION THREAD

**Discussion by kimminw00** [apache/airflow#38668] [Q&A]
```python
import datetime

from airflow import DAG
from airflow.operators.empty import EmptyOperator

with DAG(
    dag_id="my_dag_name",
    start_date=datetime.datetime(2021, 1, 1),
    owner_links={"aaa": "https://airflow.apache.org"}, # Added
    schedule="@daily",
):
    EmptyOperator(task_id="task")
```

I recently added <code>owner_links</code> to my DAG.
However, in the webserver UI, the owner value is not changed.
Is there anyone experiencing the same issue?

Enviornments
Airflow 2.7.2

**Comment by fredthomsen** [comment]
Did you ever find the root cause to this?  I'm encountering the same issue and I'm a bit confused.

> ```python
> import datetime
> 
> from airflow import DAG
> from airflow.operators.empty import EmptyOperator
> 
> with DAG(
>     dag_id="my_dag_name",
>     start_date=datetime.datetime(2021, 1, 1),
>     owner_links={"aaa": "https://airflow.apache.org"}, # Added
>     schedule="@daily",
> ):
>     EmptyOperator(task_id="task")
> ```
> 
> I recently added <code>owner_links</code> to my DAG.
> However, in the webserver UI, the owner value is not changed.
> Is there anyone experiencing the same issue?
> 
> Enviornments
> Airflow 2.7.2
